### PR TITLE
change search events from keydown to keyup

### PR DESCRIPTION
**Pull request by StephenFluin**
Fixes a bug where the last character you type isn't used for searching.

### CODE REVIEW

File: _includes/_search.html [hunk 1]
```diff
@@ -156,7 +156,7 @@
       clearTimeout(searchTimeout);
     }
       searchTimeout = setTimeout(() => {
-        console.log('timeout is activating!');
+        // console.log('timeout is activating!');
```
Comment (alexroan) [inline _includes/_search.html:159]: Better to remove these lines than comment out.

File: _includes/_search.html [hunk 2]
```diff
@@ -166,7 +166,7 @@
         SULX.post(path, JSON.stringify(data));
         searchTimeout = null;
       },1000);
-      console.log(searchTimeout);
+      // console.log(searchTimeout);
```
Comment (alexroan) [inline _includes/_search.html:169]: Better to remove these lines than comment out.

File: _includes/_search.html [hunk 3]
```diff
@@ -109,7 +109,7 @@
  
   let searchTimeout;
   const performSearch = (event) => {
-      // console.log('perform search event',event);
+      console.log('perform search event',event,document.getElementById('search-query').value);
```
Comment (alexroan) [inline _includes/_search.html:112]: Is this needed?

**Review by alexroan — CHANGES_REQUESTED** [review]

**Comment by StephenFluin** [comment]
@alexroan great call, dead code removed.

**Review by alexroan — APPROVED** [review]

**Pull request merged**